package.json

```diff
@@ -6,7 +6,7 @@
   "license": "BSD-2-Clause",
   "version": "5.3.6",
   "engines": {
-    "node": "^10.0.0,^11.0.0,^12.0.0,>=14.0.0"
+    "node": "^10.0.0 ^11.0.0 ^12.0.0 >=14.0.0"
```

Comment:
If I understand it correctly, space means "AND", so this semver query translates to "satisfy versions ^10 AND ^11 AND ^12 AND >=14" which is physically impossible, so no version will be satisfied

Reply:
Spaces should be replaced with `||` :-)

Reply:
...and the package version should be upped to "5.3.7"

Reply:
> ...and the package version should be upped to "5.3.7"

They'll probably do it in a separate commit ([like with 5.3.6](https://github.com/terser/terser/commit/571fd998ddac312bae2e987aa622d49b11d6c97d)), but still a valid point :D

Reply:
> > ...and the package version should be upped to "5.3.7"
> 
> They'll probably do it in a separate commit ([like with 5.3.6](https://github.com/terser/terser/commit/571fd998ddac312bae2e987aa622d49b11d6c97d)), but still a valid point :D

not the first time, I see :)

Reply:
Isn't this the same as just saying `>=10.0.0`? What's up with version 13?

Reply:
I have heard some unflattering comments towards Node v13 because of its bugginess. Perhaps it's the reason here. I personally never use non-LTS versions, so I can't really say if it's true or not :/